### PR TITLE
Content added, fixed, moved; things prettified (a bit)

### PR DESCRIPTION
**Pull request by DeckOfPandas**

### CODE REVIEW

File: _posts/events/2014-05-24-london.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ date_range:
 categories: events
 label: 2014
 background-image: “/assets/imgs/locations/London.jpg"
```
Comment (lamby) [inline _posts/events/2014-05-24-london.md:12]: > background-image: “/assets/imgs/locations/London.jpg"

Has a "smart quote" ie `“` when it should be "".

File: _posts/events/2015-01-24-cardiff.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ date_range:
 categories: events
 label: 2015
 background-image: “/assets/imgs/locations/Cardiff.jpg"
```
Comment (lamby) [inline _posts/events/2015-01-24-cardiff.md:12]: smart quote

File: _posts/events/2014-05-24-london.md (hunk 1 above)
Comment (lamby) [inline _posts/events/2014-05-24-london.md:12]: (I think..)

**Comment by DeckOfPandas** [comment]
Oh my goodness, you're amazing @lamby, I would literally never have spotted that! I'll have some time later today to fix this then, and all our gorgeous images will display...woo

**Comment by DeckOfPandas** [comment]
I'm going to merge this, although am aware of the bug described in issue #71

**Pull request merged**